jenkins/check-for-build.jenkinsfile

```diff
@@ -0,0 +1,89 @@
+lib = library(identifier: "jenkins@20211123", retriever: legacySCM(scm))
+
+pipeline {
+    agent none
+    triggers {
+        parameterizedCron '''
+            H/10 * * * * %INPUT_MANIFEST=1.3.0/opensearch-1.3.0.yml;TARGET_JOB_NAME=distribution-build-opensearch
+            H/10 * * * * %INPUT_MANIFEST=1.3.0/opensearch-dashboards-1.3.0.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards
```

Comment:
Did we change `bundle-build` job name to `distribution-build-opensearch`?

Reply:
Seems like as good a time as any to fix the names, this will need to be 2PR'ed on the Jenkins server to re-align all of the names, I am open to other names schemes if you have thoughts.

Reply:
I am okay with this name. With that said, do we need to make changes to the tests here: https://github.com/opensearch-project/opensearch-build/pull/1254/files#diff-fecdf689e56f633fe0618499df897827d0c7c8426fbc7a7fd0b9e9a656760fb7R24
Many tests contain `bundle-build` as their job name param

Reply:
A lot of the hardcoding and even the current 1.1.1 are linked to bundle-build as a name on S3.
Adding this change would cause a lot of build failures.

Reply:
Darn, lets follow up on a seperate PR to that removes the word bundle entirely

Reply:
Agreed! Thanks

Reply:
> A lot of the hardcoding and even the current 1.1.1 are linked to bundle-build as a name on S3.
Adding this change would cause a lot of build failures.

@peterzhuamazon This does not update any of the previous manifests or artifacts, those will stay named exactly as they are, this will change the names / paths for new builds which shouldn't have the work 'bundle' in them per https://github.com/opensearch-project/opensearch-build/issues/1132

Reply:
Are you changing the `TARGET_JOB_NAME` back to `bundle-build` since now renaming will be done entirely in new PR?

Reply:
Renaming will happen entirely on Jenkins, no new PRs will be created.  This change requires a deployment step after it has merged to create the new job and rename the old ones